### PR TITLE
Beven Nyamande

### PR DESCRIPTION
**Pull request by bevennyamande**
Fixes #

## Proposed Changes

  - Added my portfolio

### CODE REVIEW

**Review by adityaraute — COMMENTED** [review]
Wait, why are you adding it if it's not your work?

**Comment by bevennyamande** [comment]
# Edit
- Removed the dummy file and added my work, stuff i learnt through Udemy
-  A game background design

**Comment by adityaraute** [comment]
But the svg file is not linked in your readme...?

Instead some `skeleton.png` is linked, which you have mentioned as 
> Not my work

**Comment by bevennyamande** [comment]
Thanks had forgotten about that

**Review by adityaraute — APPROVED** [review]

File: 2021/01/localhackday/Beven_Nyamande/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## About
+
+- I will add more assets here
+
+#### Disclaimer
+
+- skeleton.png is not my work
```
Comment (adityaraute) [inline 2021/01/localhackday/Beven_Nyamande/README.md]: Why does this line still exist?

Comment (bevennyamande) [reply 2021/01/localhackday/Beven_Nyamande/README.md]: ops let me change it

**Review by adityaraute — APPROVED** [review]

**Comment by AllanRegush** [comment]
Is this from a hackathon or a udemy course?

**Comment by adityaraute** [comment]
Hackathon, Local Hack Day by MLH.

**Comment by adityaraute** [comment]
Why did you close it?

**Comment by bevennyamande** [comment]
I sorted all the information

**Comment by adityaraute** [comment]
SO now the PR is ready to be merged, right?

**Comment by bevennyamande** [comment]
yes you can merge

**Comment by adityaraute** [comment]
I cannot merge. A maintainer has the rights. Wait up till a maintainer merges it.

**Comment by bevennyamande** [comment]
ok thanks

**Comment by eddiejaoude** [comment]
Sorry for the delay, great collaboration 👍

**Pull request merged**